server/src/rooms/BattleRoom.ts

```diff
@@ -120,28 +122,46 @@ export class BattleRoom extends Room<BattleRoomState> {
   }
 
   resetPlayersPositions() {
-    //move the positions of all clients to the start position
     for (let team of this.state.teams) {
       for (let [playerId, inBattlePlayer] of team.teamPlayers.entries()) {
         if (inBattlePlayer != undefined) {
           // different starting position got players from different teams
           let player: Player = this.state.players.get(playerId);
           if (player != undefined) {
             if (inBattlePlayer.teamColor == TeamColor.Red) {
-              {
-                player.x = this.team_A_start_x_pos;
-                player.y = this.team_A_start_y_pos;
-              }
+              player.x = this.team_A_start_x_pos;
+              player.y = this.team_A_start_y_pos;
             } else {
               player.x = this.team_B_start_x_pos;
               player.y = this.team_B_start_y_pos;
             }
+
+            // Find the client associated with the session ID
+            const client = this.clients.find(client => client.sessionId === player.sessionId);
+
+            // Send the new position to the client
+            if (client) {
+              this.send(client, "roundStart", { x: player.x, y: player.y });
+            }
           }
         }
       }
     }
   }
 
+  private broadcastSpawnMonsters() {
+    //put monster into map, create new monster given the number
+    for (let i = 0; i < this.NUM_MONSTERS; i++) {
+      let monster = new Monster();
+      monster.x = Math.floor(Math.random() * 800);
```

Comment:
This is the resolution of the screen only though, is it possible to use the dimension of the map + some offset for this?

Reply:
Yea the thing is I hardcoded some stuff about location since I'm going to change map very soon. Maybe I'll change map in this issue itself.

Reply:
For the animations, I think it is unnecessary right now? Since we don't want our monster to be moving as randomly as the current lizards.